webdriver/bidi/emulation.json

```
@@ -204,7 +204,10 @@
               "firefox": {
                 "version_added": "142",
                 "partial_implementation": true,
-                "notes": "Both \"navigator.languages\" and the HTTP \"Accept-Language\" request header are not set yet."
+                "notes": [
+                  "Before Firefox 144 the locale override was shared with all the other browsing contexts living in the same web content process.",
+                  "Both \"navigator.languages\" and the HTTP \"Accept-Language\" request header are not set yet."
```

@whimboo Nits: MDN prefers oxford comma, and we'd probably want to use backticks instead of quotes for those.

```suggestion
                  "Before Firefox 144, the locale override was shared with all the other browsing contexts living in the same web content process.",
                  "Both `navigator.languages`, and the HTTP `Accept-Language` request header are not set yet."
```

---

Yes, sounds good. I've updated the Google spreadsheet.